ntripclient.c

```diff
@@ -566,6 +583,104 @@ static int encode(char *buf, int size, const char *user, const char *pwd)
   return bytes;
 }
 
+static ssize_t curl_send_(CURL *curl, const void *buf, size_t len, int flags)
+{
+    (void)flags; // not used
+    size_t bytes_sent = 0;
+    CURLcode result = curl_easy_send(curl, buf, len, &bytes_sent);
+    return (result == CURLE_OK) ? (ssize_t)bytes_sent : -1;
```

Comment:
You could also use the suggested `ReadBufferRetCode` enum here as well for invalid bytes sent

Reply:
This one I will want to keep the c convention for fucntions like read/write/recv/send which returns -1 on error.